genesyscloud/resource_genesyscloud_integration.go

```diff
@@ -0,0 +1,495 @@
+package genesyscloud
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/platformclientv2"
+)
+
+var (
+	integrationConfigResource = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "Integration config name.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"notes": {
+				Description: "Integration config notes.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"properties": {
```

Comment:
Do the properties and advanced attributes properly handle diffs when the config JSON string doesn't actually change (whitespace, attribute order, etc)? You may want to write a test case to verify. There is a DiffSuppress method `suppressEquivalentJsonDiffs` I added recently for JSON attributes that may help.

Reply:
Added diffsuppress function, and another testcase to test Raw JSON string